src/components/Template/hooks/useNavigation/useNavigation.ts

```diff
@@ -33,6 +34,10 @@ export const useNavigation = () => {
     {
       path: '/smart-contract',
       icon: faFile
+    },
+    {
+      path: '/explainer',
```

Comment:
use routeNames for paths

Reply:
I followed the current pattern when adding the route. I couldn't locate any **routeNames** in the project. Could you provide an example or clarify your suggestion?

Reply:
Hey @levente-daczo , please fetch the latest changes from the `development` branch. There is a const in routes.ts file. It was introduced recently.